src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        int randomIndexOfColor = new Random().nextInt(Colors.values().length);
```

Comment:
Don't create new Random every time you call this method. It's better to make Random a field of the class.
```suggestion
        int randomIndexOfColor = random.nextInt(Colors.values().length);
```